sbot/motor_board.py

```diff
@@ -0,0 +1,294 @@
+        for port in serial_ports:
+            # Filter to USB vendor and product ID of the FTDI FT232R
+            # chip used on the motor board
+            if port.vid == 0x0403 and port.pid == 0x6001:
+                # Create board identity from USB port info
+                initial_identity = get_USB_identity(port)
+
+                try:
+                    board = MotorBoard(port.device, initial_identity)
+                except BoardDisconnectionError:
+                    logger.warning(
+                        f"Found motor board-like serial port at {port.device!r}, "
+                        "but it could not be identified. Ignoring this device")
+                    continue
+                except IncorrectBoardError as err:
+                    logger.warning(
+                        f"Board returned type {err.returned_type!r}, "
+                        f"expected {err.expected_type!r}. Ignoring this device")
+                    continue
+                boards[board._identity.asset_tag] = board
+
+        # Add any manually specified boards
+        if isinstance(manual_boards, list):
+            for manual_port in manual_boards:
+                # Create board identity from the info given
+                initial_identity = BoardIdentity(
+                    board_type='manual',
+                    asset_tag=manual_port,
+                )
+
+                try:
+                    board = MotorBoard(manual_port, initial_identity)
+                except BoardDisconnectionError:
+                    logger.warning(
+                        f"Manually specified motor board at port {manual_port!r}, "
+                        "could not be identified. Ignoring this device")
+                    continue
+                except IncorrectBoardError as err:
+                    logger.warning(
+                        f"Board returned type {err.returned_type!r}, "
+                        f"expected {err.expected_type!r}. Ignoring this device")
+                    continue
+                boards[board._identity.asset_tag] = board
+        return MappingProxyType(boards)
+
+    @property
+    @log_to_debug
+    def motors(self) -> tuple[Motor, Motor]:
+        """
+        A tuple of the two motors on the board.
+
+        :return: A tuple of the two motors on the board.
+        """
+        return self._motors
+
+    @log_to_debug
+    def identify(self) -> BoardIdentity:
+        """
+        Get the identity of the board.
+
+        :return: The identity of the board.
+        """
+        response = self._serial.query('*IDN?')
+        return BoardIdentity(*response.split(':'))
+
+    @property
+    @log_to_debug
+    def input_voltage(self) -> float:
+        """
+        The input voltage to the board.
+
+        :return: The input voltage to the board.
+        """
+        response = self._serial.query('*STATUS?')
+        return MotorStatus.from_status_response(response).input_voltage
```

Comment:
Suggestion: I think we should keep the generic `status` property to retrieve the unified status data, and perhaps we can just use `board.status.input_voltage` then.

Reply:
Yeah, that works.

5f3fdd3aea32367ce63b28ca8bc081bec6a40285